### PR TITLE
老师，做完了

### PR DESCRIPTION
**Pull request by ZYT-zyt**

### CODE REVIEW

File: Phrase Template.java [hunk 1]
```diff
@@ -16,8 +16,18 @@ public Phrase(String p)
...
+            if(i == -1)
+                return -1;
+            else if(n == 1)
+                index += i;
+            else
+                index += i + str.length();
+            n--;
+        }
+        return index;
     }
```
Comment (Jack-Starship) [inline Phrase Template.java:31]: Think about the following example
try to find the third "aa" from "aabaaa"

**Comment by ZYT-zyt** [comment]
so，I just replace index +length + i to index + 1 + i in line 27 @Jack-Starship

**Comment by ZYT-zyt** [comment]
so，I just replace index +length + i to index + 1 + i in line 27 @Jack-Starship

**Comment by ZYT-zyt** [comment]
so，I just replace index +length + i to index + 1 + i in line 27 @Jack-Starship

**Pull request merged**